src/channel_server/player_handler.cpp

```diff
@@ -1088,11 +1094,14 @@ auto player_handler::compile_attack(ref_ptr<player> player, packet_reader &reade
 
 	for (int8_t i = 0; i < targets; ++i) {
 		game_map_object map_mob_id = reader.get<game_map_object>();
-		reader.unk<uint32_t>(); // Always 0x06, <two bytes of some kind>, 0x01
+		reader.skip<int8_t>(); // Hit action
+		reader.skip<int8_t>(); // 'Fore Action' (where bit 8 == facing left)
+		reader.skip<int8_t>(); // Frame index
+		reader.skip<int8_t>(); // Damage stats calculator index. Bit 8 == mob doomed
```

Comment:
If any of these don't allow negative values, I would recommend `uint8_t` in their place. I assume bitfields that have bit 8, for example, should be `uint8_t`. Also what are things like "hit action"? Does that refer to a .wz animation? More descriptive commenting would be helpful.